js/__internal/grids/grid_core/column_headers/m_column_headers.ts

```diff
@@ -571,7 +571,10 @@ export const columnHeadersModule = {
             rowIndex = rowIndex || 0;
             for (i = rowIndex; i < rowCount; i++) {
               columnElements = that.getCellElements(i);
-              each(that.getColumns(i), setColumnOpacity);
+
+              if (columnElements) {
+                each(that.getColumns(i), setColumnOpacity);
```

Comment:
Let's no use ```each``` here (it's [banned by liner](https://github.com/DevExpress/DevExtreme/blob/23_1/js/__internal/.eslintrc.js#L45), but there is only warning in old files)
Also, let's change ```that``` -> ```this``` in this method.

We can do something like:

```suggestion
                const columns = this.getColumns(i);
                columns.forEach((columnItem, idx) => setColumnOpacity(columnItem, idx));
                // Or if we swap setColumnOpacity arguments, we can use it this way:
                columns.forEach(setColumnOpacity);
```